### ISSUE TITLE
Uninstall-ChocolateyZipPackage not cleaning up due to incorrectly named manifest txt file

### ISSUE THREAD

**Issue by tomsie1000** [chocolatey/choco#1520]
### What You Are Seeing?

When uninstalling Chocolatey is supposed to clean up the contents of an unzipped portable package by keeping a manifest file of the unzipped package contents, including the path. The files are not being removed during the uninstall process.

I believe this was detailed by another user @berndb84 on issue https://github.com/chocolatey/choco/issues/127 (but only after it was closed)

### How Did You Get This To Happen? (Steps to Reproduce)

Let's assume a portable package of `test.portable` which installs from a locally-hosted zip file `test.portable.zip`

As part of my `ChocolateyInstall.ps1` I call `Install-ChocolateyZipPackage @packageArgs`

`$packageArgs = @{`
`  packageName   = $env:ChocolateyPackageName`
`  unzipLocation = $env:SystemDrive + "\test"`
`  file         = 'D:\chocolatey\local-repo\test.portable.zip'`
`}`
`Install-ChocolateyZipPackage @packageArgs`

During the install the `Get-ChocolateyUnzip` function creates a .txt file in the package lib folder - in this instance it creates `test.portableInstall.zip.txt`. **This may in itself be the issue** - the string 'Install' is added which confuses the next step.

Now I call `choco uninstall test.portable`, and in my `ChocolateyUninstall.ps1` file I have `Uninstall-ChocolateyZipPackage $packageName "test.portable.zip"`. The contents of the folder at `C:\Test` are not removed.

My `ChocolateyUninstall.ps1` has the following:

`Uninstall-ChocolateyZipPackage $packageName "test.portable.zip"`

As a workaround, the following 'fixes' the issue:

`Uninstall-ChocolateyZipPackage $packageName "test.portableInstall.zip"`

**Comment by ferventcoder** [comment]
@tomsie1000 actually, I believe this is a duplicate of #1415, which has been fixed for the 0.10.9 release.

**Comment by aventrax** [comment]
Really? I'm using the latest 0.10.15 and I packed a zip file. The file which `Install-ChocolateyZipPackage` creates with the list of extracted file is located in `C:\ProgramData\chocolatey\lib\instantclient\instantclientInstall.zip.txt`  and a normal `Uninstall-ChocolateyZipPackage   -PackageName -ZipFileName`  would remove the files on the destination folder.

The workaround from @tomsie1000 works!
